pkg/drivers/vfkit/driver_darwin.go

```diff
@@ -97,20 +97,21 @@ func (d *Driver) getDiskPath() string {
 	return d.ResolveStorePath(fmt.Sprintf("%s.img", d.MachineName))
 }
 
-func (d *Driver) resize(newSize int64) error {
+func (d *Driver) resize(newSize uint64) error {
+	newSizeInt64 := int64(newSize) //#nosec G115 -- os.stat provide the size as int64 and os.Truncate use int64 as arg
```

Comment:
This explains why there's a warning, but this should be explaining why it's ok to ignore it. This needs range checks before doing the cast.